command/cleanup.go

```diff
@@ -60,10 +61,16 @@ func CmdCleanup(c *cli.Context) (err error) {
 
 	for _, name := range toUndeploy {
 		log.Println("Undeploying", name)
-		err := exec.Command(undeployScript, name).Run()
+		err = exec.Command(undeployScript, name).Run()
 		if err != nil {
 			log.Println("undeploy error:", err)
+			continue
+		}
+		pullRequestId, err := strconv.Atoi(name)
+		if err != nil {
+			log.Fatalf("Unable to parse pull request id: %s", name)
```

Comment:
Should this abort the loop entirely? Or just continue after logging an error like the error above.

Reply:
Indeed, well spotted